taskt/Core/Automation/Commands/Error Handling Commands/FinallyCommand.cs

```diff
@@ -1,37 +1,38 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Windows.Forms;
-using System.Xml.Serialization;
-using taskt.Core.Automation.Engine;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Script;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Error Handling Commands")]
-    [Attributes.ClassAttributes.Description("This command specifies execution that should occur whether or not an error occured")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to always execute a specific command before leaving the try/catch block")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Error Handling Commands")]
+    [Description("This command specifies execution that should occur whether or not an error occured")]
```

Comment:
Please use punctuation in **Description**

Reply:
Done.